packages/frontend/core/src/desktop/pages/workspace/index.tsx

```diff
@@ -110,6 +117,17 @@ export const Component = (): ReactElement => {
   }, [listLoading, meta, workspaceNotFound, workspacesService]);
 
   if (workspaceNotFound) {
+    if (BUILD_CONFIG.isElectron && searchParams.get('server')) {
```

Comment:
```suggestion
    if (BUILD_CONFIG.isElectron && serverNotFound) {
```

Reply:
const serversService = useService(ServersService)
const serverNotFound = serversService.getServerByBaseUrl(searchParams.get('server')))

Reply:
It is possible that the server already exists but the user has not logged in.

Reply:
It also be possible that the user logged in with the wrong account